### PR TITLE
success

### PR DESCRIPTION
**Pull request by vladisssi**
[DEMO LINK](https://vladisssi.github.io/react_people-table-advanced/)

### CODE REVIEW

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, your demo link isn't working, you need to make a deploy and check it after on the demo 
image

File: src/components/PeopleFilters.tsx [hunk 1]
```diff
@@ -1,18 +1,70 @@
...
+import { useSearchParams } from 'react-router-dom';
+
+export enum Status {
+  All = '',
+  Male = 'm',
+  Female = 'f',
+}
+
 export const PeopleFilters = () => {
+  const centuriesArray = ['16', '17', '18', '19', '20'];
```
Comment (volodymyr-soltys97) [inline src/components/PeopleFilters.tsx:14]: This array needs to be memorized

File: src/utils/ways.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export enum WAYS {
```
Comment (volodymyr-soltys97) [inline src/utils/ways.ts:1]:
```suggestion
export enum ERoutes {
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
1. Don't need to reset filters after clicking on the person 
image
image
image

2. Clear the selected people after reset all filters 
image

**Comment by vladisssi** [comment]
in demo link we need to reset filters after clicking on the person

File: src/utils/filterPeople.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+  if (sort) {
+    preparedPeople.sort((a, b) => {
+      switch (sort) {
+        case 'name':
+          return a[sort].localeCompare(b[sort]);
+
+        case 'sex':
+          return a[sort].localeCompare(b[sort]);
+
```
Comment (etojeDenys) [inline src/utils/filterPeople.ts:49]:
```suggestion
        case 'name':
        case 'sex':
          return a[sort].localeCompare(b[sort]);

```
you can combine cases

File: src/utils/filterPeople.ts [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        case 'born':
+          return a[sort] - b[sort];
+
+        case 'died':
+          return a[sort] - b[sort];
```
Comment (etojeDenys) [inline src/utils/filterPeople.ts:54]: same here

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Not fixed 1st point  from the [comment](https://github.com/mate-academy/react_people-table-advanced/pull/1054#pullrequestreview-2293197759)

after click on person filters shouldn't reset

https://github.com/user-attachments/assets/554a51e0-9b77-4d80-9532-655659d51ef7

**Pull request closed without merging**